modules/widgets/twitter-timeline.php

```diff
@@ -440,7 +447,7 @@ class="widefat"
 				type="checkbox"<?php checked( in_array( 'noscrollbar', $instance['chrome'] ) ); ?>
 				id="<?php echo $this->get_field_id( 'chrome-noscrollbar' ); ?>"
 				name="<?php echo $this->get_field_name( 'chrome' ); ?>[]"
-				value="noscrollbar"
+				value="noscrollbar"<?php disabled( strlen( strval( $instance['tweet-limit'] ) ) > 0 ); ?>
```

Comment:
Do you think we could simplify this by sanitizing `$instance['tweet-limit']` earlier?

Reply:
Updated in 709a38758e3d9d9ee7bf15d22f3b7b7041737e57